### PR TITLE
checkbashisms: update to 2.18.2.

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: srcpkgs/checkbashisms/template [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+checkdepends="perl shunit2"
 short_desc="Debian script that checks for bash-isms"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
-homepage="https://tracker.debian.org/pkg/devscripts"
 license="GPL-2.0-or-later"
+homepage="https://tracker.debian.org/pkg/devscripts"
 distfiles="${DEBIAN_SITE}/main/d/devscripts/devscripts_${version}.tar.xz"
-checksum=96886bf5d6605a619a12d56b9af5eb354106bf754d9b0fd562fbcbb2bf7b5ef9
+checksum=645b5814be7a9d1f4f3df1931f6d9c645e1fd75071ddafabd798f9459277d435
+noarch=yes
```
Comment (asergi) [inline srcpkgs/checkbashisms/template]: Please no `noarch=yes` at the bottom. People always forget to add it, if we move it to the bottom on purpose...

Comment (asergi) [reply srcpkgs/checkbashisms/template]: Also, you can drop all other changes, but `version` and `checksum`.

**Comment by maxice8** [comment]
On Mon, Apr 23, 2018 at 12:42:39AM -0700, Alessio Sergi wrote:
> @asergi commented on this pull request.
> 
> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
> 
> In srcpkgs/checkbashisms/template:
> 
> >  distfiles="${DEBIAN_SITE}/main/d/devscripts/devscripts_${version}.tar.xz"
> -checksum=96886bf5d6605a619a12d56b9af5eb354106bf754d9b0fd562fbcbb2bf7b5ef9
> +checksum=645b5814be7a9d1f4f3df1931f6d9c645e1fd75071ddafabd798f9459277d435
> +noarch=yes
> 
> Please no noarch=yes at the bottom. People always forget to add it, if we move
> it to the bottom on purpose...
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.*
> 

where should it be instead ?

**Comment by asergi** [comment]
Usually after `revision`.

**Comment by maxice8** [comment]
On Mon, Apr 23, 2018 at 08:13:42AM +0000, Alessio Sergi wrote:
> Usually after revision.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.*
> 

alright, i'll add to style.7

**Pull request merged**